src/components/MapView/DateSelector/index.tsx

```diff
@@ -172,6 +356,87 @@ const styles = (theme: Theme) =>
     divider: {
       backgroundColor: theme.palette.grey[500],
     },
+
+    dateContainer: {
+      position: 'relative',
+      height: '36px',
+      flexGrow: 1,
+      cursor: 'e-resize',
+      overflow: 'hidden',
+    },
+
+    dateLabelContainer: {
+      position: 'absolute',
+      flexWrap: 'nowrap',
+    },
+
+    timeline: {
+      position: 'relative',
+      top: '5px',
+    },
+
+    dateItemFull: {
+      borderLeft: '1px solid white',
+      height: '36px',
```

Comment:
using theme.spacing would be nice where possible. Also, these could just be numbers if you are using pixel measurements.